h5py/h5t.pyx

```diff
@@ -28,17 +28,23 @@ from .utils cimport  emalloc, efree, require_tuple, convert_dims,\
 
 # Python imports
 import codecs
+import os
 import sys
 from collections import namedtuple
-import sys
 import numpy as np
 from .h5 import get_config
 
 from ._objects import phil, with_phil
 
 cfg = get_config()
 
-DEF MACHINE = UNAME_MACHINE  # processor architecture, provided by Cython
+if sys.platform.startswith("win"):
```

Comment:
note to reviewers: this branch exists to get around `os.uname` not being available outside UNIX